code/modules/mob/living/simple_animal/friendly/drone.dm

```diff
@@ -607,3 +608,21 @@
 	icon_state = "syndrone_item"
 	drone_type = /mob/living/simple_animal/drone/syndrone
 
+
+// malf drone
+/mob/living/simple_animal/drone/syndrone/malf
```

Comment:
Don't worry, I'll update this. Thinking of using the default 30 health and a different sprite. Even debating if they should start with a toolbox or not.